site/scripts/prepare-blocks.ts

```diff
@@ -174,7 +174,6 @@ const ensureRepositorySnapshot = async ({
         "--extract",
         `--file=${path.resolve(tarDirPath, "repo.tar.gz")}`,
         `--directory=${outputDirPath}`,
-        "--verbose",
```

Comment:
We discussed this flag with @akash-joshi in DMs.

This change improves Vercel logs. Before:
![image](https://user-images.githubusercontent.com/608862/157898401-c0afa1ab-1284-4317-ac8c-26dda72493f1.png)